plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -78,6 +78,10 @@
   protected static final long ROCKSDB_BLOCKCACHE_SIZE_HIGH_SPEC = 1_073_741_824L;
   /** RocksDb memtable size when using the high spec option */
   protected static final long ROCKSDB_MEMTABLE_SIZE_HIGH_SPEC = 1_073_741_824L;
+  /** RocksDb number of log files to keep on disk */
+  private static final long NUMBER_OF_LOG_FILES_TO_KEEP = 5;
+  /** RocksDb Max size of each log file */
+  private static final long MAX_LOG_FILE_SIZE = 104_857_600L;
```

Comment:
Does this mean we will always fill up 500MB worth of logs if the node has been up for long enough, or might the files get rotated for reasons other than hitting the 100MB size?

Reply:
From what I noticed, there is a new log file for each restart.